setup.py

```diff
@@ -1,36 +1,35 @@
-import io
 import os
-from setuptools import setup, find_packages
 
 import versioneer
+from setuptools import find_packages, setup
 
 PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
 
-DESCRIPTION = 'Python module and CLI for hashing of file system directories.'
+DESCRIPTION = "Python module and CLI for hashing of file system directories."
 
 try:
-    with io.open(os.path.join(PROJECT_ROOT, 'README.md'), encoding='utf-8') as f:
-        long_description = '\n' + f.read()
-except IOError:
+    with open(os.path.join(PROJECT_ROOT, "README.md"), encoding="utf-8") as f:
+        long_description = "\n" + f.read()
+except OSError:
     long_description = DESCRIPTION
 
 setup(
-    name='dirhash',
+    name="dirhash",
     version=versioneer.get_version(),
     cmdclass=versioneer.get_cmdclass(),
     description=DESCRIPTION,
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url='https://github.com/andhus/dirhash-python',
+    url="https://github.com/andhus/dirhash-python",
     author="Anders Huss",
     author_email="andhus@kth.se",
-    license='MIT',
-    install_requires=['scantree>=0.0.2'],
```

Comment:
manual change - loosening scantree range

Reply:
Sure :) There is really no reason to be below that version, but strictly speaking yes, there should be nothing public-api-wise needed from 0.0.2 to get dirhash to run.